packages/calcite-design-tokens/src/semantic/font.json

```diff
@@ -663,254 +663,6 @@
             }
           }
         }
-      },
-      "letter-spacing": {
```

Comment:
Would this be breaking due to these removals?

Reply:
technically yes but they were not used by any components.

Reply:
They are mentioned in the design token [reference doc](https://developers.arcgis.com/calcite-design-system/foundations/tokens/reference/).

Reply:
tokens are updated to be deprecated only